homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -16,8 +300,12 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     async def async_step_import(self, import_config=None):
         """Handle the initial step."""
-        entry = await self.async_set_unique_id(DOMAIN)
-        if entry and import_config.items() != entry.data.items():
-            self.hass.config_entries.async_update_entry(entry, data=import_config)
-            return self.async_abort(reason="already_configured")
+        await self.async_set_unique_id(DOMAIN)
```

Comment:
I just realized we probably need migration anyway and up the version of the ConfigEntry.

Users with 0.113 have no data at all stored in the ConfigEntry, and if they want to upgrade to a release where this merges in, the ConfigEntry will not be updated from yaml. That's quite user unfriendly.

Reply:
Only if we stop importing. Right now we re-write then config entry stored based on yaml. As soon as we turn it into a one time import yes.

We could during the first version that imports once, check if the stored entry is an old version and just ignore it completely then. Since all data should have come from yaml in that case and we can just do a full import.